samples/react-sample-app/src/components/NavBar.tsx

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from "react";
+import AppBar from "@mui/material/AppBar";
+import { IconButton, Toolbar } from "@mui/material";
+import Typography from "@mui/material/Typography";
+import AccountCircle from '@mui/icons-material/AccountCircle';
+import MenuItem from '@mui/material/MenuItem';
+import Menu from '@mui/material/Menu';
+import { useAuthContext } from "@asgardeo/auth-react";
+
+export const NavBar = () => {
+    const { state, signOut } = useAuthContext();
+    const [ anchorEl, setAnchorEl ] = React.useState<null | HTMLElement>(null);
+
+    const handleMenu = (event: React.MouseEvent<HTMLElement>) => {
+        setAnchorEl(event.currentTarget);
+    };
+
+    const handleClose = () => {
+        setAnchorEl(null);
+    };
+
+    const handleLogout = () => {
+        signOut().catch((e) => console.log("Something went wrong while signing out. ", e));
```

Comment:
Remove unnecessary console logs. And let's handle the error properly.

Reply:
Resolved at https://github.com/wso2-extensions/identity-verification-onfido/pull/17